pkgs/servers/monitoring/prometheus/exportarr/default.nix

```
@@ -12,22 +12,22 @@ buildGoModule rec {
     owner = "onedr0p";
     repo = "exportarr";
     rev = "v${version}";
-    hash = "sha256-KTuOhyBFS6fgA9N70vq+5fJIGVsgEZ7Uxls8efqLuII=";
+    sha256 = "sha256-KTuOhyBFS6fgA9N70vq+5fJIGVsgEZ7Uxls8efqLuII=";
```

```suggestion
    hash = "sha256-KTuOhyBFS6fgA9N70vq+5fJIGVsgEZ7Uxls8efqLuII=";
```

No reason to change the attribute name, when you're putting in an SRI hash anyway.

---

I agree, but I'm currently hitting a [bug](https://github.com/Mic92/nix-update/issues/80#issuecomment-1237875667) with `nix-update`/`fetchurl` which breaks automatic updating if `hash` is specified instead of `sha256`. So, I decided to put `sha256` there.